### PR TITLE
[TR] PIM-4429: Re-enable / re-write skip-pef scenario

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4429

EE PR : https://github.com/akeneo/pim-enterprise-dev/pull/929

### CODE REVIEW

File: features/Context/Page/Product/Edit.php [hunk 1]
```diff
@@ -60,25 +61,33 @@ public function verifyAfterLogin()
...
      * @param string $locale locale label
      * @param string $flag   class of the flag icon
+     * @param bool   $copy   search in copy panel or main panel
      *
      * @throws ElementNotFoundException
      *
      * @return NodeElement|null
      */
-    public function findLocaleLink($locale, $label, $flag = null)
+    public function findLocaleLink($locale, $label = null, $flag = null, $copy = false)
```
Comment (jjanvier) [inline features/Context/Page/Product/Edit.php]: typehint locale or name it localeCode

File: features/Context/Page/Product/Edit.php [hunk 2]
```diff
@@ -1116,46 +1107,31 @@ public function findCategoryInTree($category)
...
     }
 
     /**
-     * @param string $language
+     * @param string $locale
      *
      * @throws \InvalidArgumentException
      */
-    public function compareWith($language)
+    public function compareWith($locale)
```
Comment (jjanvier) [inline features/Context/Page/Product/Edit.php]: same here

File: features/Context/Page/Product/Edit.php [hunk 3]
```diff
@@ -1184,12 +1156,19 @@ public function autoSelectTranslations($mode)
     /**
      * Manually select translation given the specified field label
      *
-     * @param string $field
+     * @param string $name
+     *
+     * @throws ElementNotFoundException
      */
-    public function manualSelectTranslation($field)
+    public function manualSelectTranslation($name)
```
Comment (jjanvier) [inline features/Context/Page/Product/Edit.php]: fieldLabel ? according to your phpdoc

**Comment by nidup** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

**Pull request merged**